packages/mdc-list/index.js

```diff
@@ -234,6 +229,26 @@ class MDCList extends MDCComponent {
           listItem.click();
         }
       },
+      toggleCheckbox: (index) => {
+        const listItem = this.listElements[index];
+        const elementsToToggle =
+          [].slice.call(listItem.querySelectorAll('input[type="radio"], input[type="checkbox"]'));
+        elementsToToggle.forEach((element) => {
```

Comment:
If we `let` a boolean outside of this loop and set it to true when this function is run, we can return it as an indicator of whether an input was found to toggle, and `preventDefault` on the event within the foundation if this returns true.